functions.yml

```diff
@@ -335,6 +351,7 @@ quality_control:
           doc: The reference allele
         - name: alternateAlleles
           doc: An array of alternate alleles
+      returns: A struct containing ``sampleId``, ``callRate``, ``nCalled``, ``nUncalled``, ``nHomRef``, ``nHet``, ``nHomVar``, ``nSnp``, ``nInsertion``, ``nDeletion``, ``nTransition``, ``nTransversion``, ``nSpanningDeletion``, ``rTiTv``, ``rInsertionDeletion``, ``rHetHomVar`` fields. See :ref:`sample-qc`.
     - name: sample_dp_summary_stats
       doc: Compute per-sample summary statistics about the depth field in an array of genotype structs
```

Comment:
`Compute per-sample summary statistics about the depth field in an array of genotype structs` -> `Computes per-sample summary statistics about the depth field in an array of genotype structs.`

Reply:
Done